vita3k/kernel/src/thread/thread.cpp

```diff
@@ -172,7 +172,7 @@ int start_thread(KernelState &kernel, const SceUID &thid, SceSize arglen, const
 
         raise_waiting_threads(thread.get());
 
-        SDL_WaitThread(running_thread, nullptr);
+        SDL_WaitThread(running_thread, &(thread->exit_status));
```

Comment:
Do we always return exit status? Apps sometimes exit the thread explicitly by sceKernelExitThread.

Reply:
Tnx, I try to implement exit status in sceKernelExitThread